snuba/cli/consumer.py

```diff
@@ -116,6 +116,13 @@
     type=int,
 )
 @click.option("--join-timeout", type=int, help="Join timeout in seconds.", default=5)
+@click.option(
+    "--enforce-schema",
+    type=bool,
+    is_flag=True,
+    default=False,
```

Comment:
Maybe a silly question, but why is this False by default?

Reply:
False is the current behaviour. Eventually we'll want enforcement to be default but we're slow rolling to reduce risk: enforcing a schema that isn't guaranteed to be correct might mean messages get incorrectly dropped.

Reply:
That makes sense. Thanks!